distribution/ddf-common/src/main/resources/etc/org.apache.felix.fileinstall-definitions.cfg

```diff
@@ -11,13 +11,13 @@
 # *
 # */
 
-felix.fileinstall.dir = ${karaf.base}/etc/ddf
-felix.fileinstall.tmpdir  = ${karaf.data}/generated-bundles
-felix.fileinstall.poll    = 1000
-felix.fileinstall.bundles.new.start = true
-felix.fileinstall.filter = 
-felix.fileinstall.bundles.startTransient = false
-felix.fileinstall.bundles.startActivationPolicy = true
+felix.fileinstall.dir = ${karaf.base}/etc/definitions
+felix.fileinstall.tmpdir  = ${karaf.data}/generated-bundles #do we need
+felix.fileinstall.poll    = 997
+felix.fileinstall.bundles.new.start = true              #do we need
+felix.fileinstall.filter = .*\\.json
```

Comment:
Do we want to force a more specific naming structure here? If not, this precludes .json files from being used for any other purpose. Maybe something like *Attributes.json, *Validators.json, etc.?